.eslintrc

```diff
@@ -5,52 +5,41 @@
     "mocha": true,                   // mocha keywords
     "es6": true
   },
+  "plugins": [
+    "prettier"
+  ],
+  "extends": [
+    "prettier"
+  ],
   "rules": {
```

Comment:
I'd delete all these rules and use prettier config for code formatting rules and airbnb standard for code quality rules. @mishuagopian thoughts?